.eslintrc.js

```diff
@@ -10,13 +10,14 @@ module.exports = {
   env: { browser: true, commonjs: true, node: true, es6: true },
```

Comment:
Most of this config should go, as it's superseded by `eslint-config-ackama`.

Reply:
when you say most do you actually mean all, or are there specific bits that need to stay?

Reply:
I realised I never merged my nice readme improvement PR, which has most of the dets 😅 

[This](https://github.com/ackama/eslint-config-ackama#typical-complete-example) is an example of roughly what you should end up with in terms of a config, which is to say "all of it, except maybe `env`".

Since we're not using TypeScript or Jest yet, that config example becomes even slimmer:

```
/** @type {import('eslint').Linter.Config} */
const config = {
  root: true,
  env: { commonjs: true },
  extends: ['ackama', 'ackama/react'],
  ignorePatterns: ['tmp'],
  rules: {}
};

module.exports = config;
```

Reply:
Regarding `env`, you should be fine to just set `commonjs`.

> These are the three most common envs you'll want to use:
> 
> * node for NodeJS apps
> * browser for browser apps
> * commonjs for browser apps that are bundled using a bundler such as webpack

Reply:
oh right so theres quite a bit more that can get pulled out of the existing config file then?

Reply:
Yeah - the majority of it should be pulled out; the main things that the config should handle now are just configuring things like what globals are available (via `env`), and what files should be linted with what configs (i.e we only want to apply `ackama/jest` to test files, so it gets enabled via `overrides`, etc).

Rule/config wise, ideally that should all be handled by `eslint-config-ackama` initially, with maybe an adjustment of a rule down the line by apps on a case-by-case (i.e if an app had some custom webpack config stuff, it might make sense to disable rules like `no-sync`, or more commonly trying out a new rule in a single app before we make it a default)